### ISSUE TITLE
Coordinates - translated values at import & export

### ISSUE THREAD

**Issue by Yan-Koch** [open-scan-crew/OpenScanTools#36]
What is expected:
When importing scans, it is possible to translate the coordinates. In case large coordinates are detected, automatic values are calculated, but they can be modified manually . The translation values also appear in the project properties (Home tab).
When exporting the point clouds, there is an option to restore the original coordinates. This should work for full scans, clipped scans, and scans clipped with gridded boxes.

However, several bugs have been detected.

1.  **Large coordinates** are detected, but the user **unchecks** the translation option (which means he wants to keep the original large coordinates):
The scans appear with its original coordinates on the screen. However, after importing the scan, the properties of the project display the proposed translation values instead of displaying 0. It is as if the project properties didn't take the unchecked status into account. This causes an issue when exporting with the "Restore original coordinates" option, as it adds these translations to the original coordinates.

Properties after importing a large coordinates scan:
![image](https://github.com/user-attachments/assets/7dd92c27-f5fd-4a7a-a1b5-80c2867df502)

After exporting and then importing again, the translation values proposed are 2x original coordinates values.
![image](https://github.com/user-attachments/assets/8572e3cc-d878-4a43-ad2b-bc44c5a68d45)

2. **Large coordinates** are detected, and the user **checks** the translation option, and **manually** changes some values:

The x value is changed before importing (x -500000)
![image](https://github.com/user-attachments/assets/96a1a669-d3e5-42bf-aea1-80817e32210e)

But the properties still shows the original value (x -633800) proposed before it was changed manually 
![image](https://github.com/user-attachments/assets/a4dd5df1-a682-4725-a41b-4def6ef6d8e3)

However, the coordinates displayed in the viewport take the manual value into account:
![image](https://github.com/user-attachments/assets/f37a1261-652a-46be-9961-94b4a175ad6a)

The project properties doesn't seem to take the manual translation values into account.

3. **Coordinates are not large** (no detection), and the user **checks** the translation option and **manually** changes some values: same as previous point. The project doesn't take the manual translation into account in the properties.

4. The **gridded box** export seems to ignore the "Restore original coordinates" option.

In summary, it appears that:
- the properties of the project don't take into account:
   - the manual values of the translations when importing,
   - the status "Inactive" when the user unchecks the translation option (in this case, the properties should display 0,0,0).
- the "Restore original coordinates" doesn't work with gridded box export.

You can use this file for testing:
https://drive.google.com/file/d/12Qn75IDi938hKPOE7shZ42LCtAMrJjFB/view?usp=drive_link

Code keywords to help solving the issue:
m_projectInfo => stores the different properties of the project, including the global translation
m_importScanTranslation
m_ui.xTruncValue
exportWithScanImportTranslation
m_scanTranslationToAdd  => maybe not used in the gridded box export
